packages/gestalt/src/Typeahead.js

```diff
@@ -135,21 +127,20 @@ const TypeaheadWithForwardRef: React$AbstractComponent<Props, HTMLInputElement>
   };
 
   // Handler for when text is typed
-  // eslint-disable-next-line no-shadow
```

Comment:
Removed this suppression, aliased arg and updated where used

Reply:
nice